### PR TITLE
New resource xSPHealthAnalyzerRuleState

### PR DESCRIPTION
**Pull request by ykuijs**
Added a new resource xSPHealthAnalyzerRuleState, issue #49 

Please review and let me know any comments

### CODE REVIEW

File: Modules/xSharePoint/DSCResources/MSFT_xSPHealthAnalyzerRuleState/MSFT_xSPHealthAnalyzerRuleState.schema.mof [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+**Description**
+
+This resource is used to set the global antivirus settings for the local farm.
+These settings will be used to control the behavior of an external anti-virus scanning tool that is able to integrate with SharePoint.
+Note that this will not scan documents for viruses on it's own, an external tool still needs to be installed on the servers that integrates with SharePoint.
```
Comment (BrianFarnhill) [inline Modules/xSharePoint/DSCResources/MSFT_xSPHealthAnalyzerRuleState/MSFT_xSPHealthAnalyzerRuleState.schema.mof]: We need to update the description here, looks like it was copied from the antivirus settings resource

**Comment by BrianFarnhill** [comment]
@ykuijs Just a quick doco fix and do a pull from dev to resolve the conflicts and then this is ready to merge.

**Comment by ykuijs** [comment]
@BrianFarnhill - I made the changes, but now the test fails due to an error in the ValidateSets. However I do not see what is wrong. Do you have any clue?

**Comment by ykuijs** [comment]
Ok, it were the spaces between the values in the ValidateSet :-s Probably an option to make the test less strict :-p

**Comment by ykuijs** [comment]
@BrianFarnhill - The tests now succeeded. Go ahead and merge it into Dev

**Pull request merged**